amr-wind/utilities/sampling/FieldNorms.H

```diff
@@ -0,0 +1,89 @@
+#ifndef FIELDNORMS_H
+#define FIELDNORMS_H
+
+#include "amr-wind/CFDSim.H"
+#include "amr-wind/utilities/PostProcessing.H"
+
+/**
+ *  \defgroup Field Norm utilities
+ *  Field Norm utilities FIXME: need to update this
```

Comment:
Is the `FIXME` still necessary? I don't think a one off utility deserves its own group. Sampling used a sub-group because it had SamplingContainer, Sampling, and different sampling probe types.